### DISCUSSION TITLE
Does a closed cavity contribute to molecular (Connolly) surface or NanoShaper SES?

### DISCUSSION THREAD

**Discussion by rwxayheee** [Electrostatics/apbs#190] [Q&A] [answered]
Dear APBS experts, 

My apologies for spamming this discussion board.. I am using "surface_method = smoothed molecular surface" for multigrid FD (mg-auto) calculation, and "mesh = NanoShaper SES" for tabi calculation. And I was wondering whether a cavity would be assigned the bulk solvent dielectric / additional triangulated surface, if that cavity is completely closed and disconnected from the outside surface. 
Is there possibly a way to write the accessibility to a .dx file? Thank you for your time and kind advice in advance. 

Many thanks,
Amy

**Comment by sobolevnrm** [comment, marked as answer]
I believe the cavities *should* be marked as solvent dielectric but would like @lwwilson1 to confirm.  Thanks.

**Reply by lwwilson1** [reply]
Yes, if NanoShaper appropriately generates the cavities, then it will effectively be treated as solvent dielectric by the boundary element procedure.

**Reply by sobolevnrm** [reply]
Thank you!
